include/boost/json/value.hpp

```diff
@@ -2830,9 +2830,135 @@ struct key_value_pair
     std::uint32_t next_;
 };
 
+//----------------------------------------------------------
+
+template <std::size_t I>
+auto
+get(key_value_pair const&) noexcept ->
```

Comment:
These are missing Javadocs. I think it would be best to ifdef all of this so it is invisible to the documentation tool, and put a generic declaration in there just for the docs. Something like this:
```
#ifdef BOOST_JSON_DOCS

/** Tuple-like interface
*/
template< std::size_t I, class T >
__see_below__
get( T&& );

#else

// actual defintions

#endif
```